metron-platform/metron-common/src/main/java/org/apache/metron/common/query/generated/PredicateLexer.java

```diff
@@ -114,59 +118,74 @@ public PredicateLexer(CharStream input) {
+		"\3\2\2\2\63\u00bf\3\2\2\2\65\u00c1\3\2\2\2\67\u00cf\3\2\2\29:\7c\2\2:"+
+		";\7p\2\2;B\7f\2\2<=\7(\2\2=B\7(\2\2>?\7C\2\2?@\7P\2\2@B\7F\2\2A9\3\2\2"+
+		"\2A<\3\2\2\2A>\3\2\2\2B\4\3\2\2\2CD\7q\2\2DJ\7t\2\2EF\7~\2\2FJ\7~\2\2"+
+		"GH\7Q\2\2HJ\7T\2\2IC\3\2\2\2IE\3\2\2\2IG\3\2\2\2J\6\3\2\2\2KL\7p\2\2L"+
+		"M\7q\2\2MR\7v\2\2NO\7P\2\2OP\7Q\2\2PR\7V\2\2QK\3\2\2\2QN\3\2\2\2R\b\3"+
+		"\2\2\2ST\7v\2\2TU\7t\2\2UV\7w\2\2V\\\7g\2\2WX\7V\2\2XY\7T\2\2YZ\7W\2\2"+
+		"Z\\\7G\2\2[S\3\2\2\2[W\3\2\2\2\\\n\3\2\2\2]^\7h\2\2^_\7c\2\2_`\7n\2\2"+
+		"`a\7u\2\2ah\7g\2\2bc\7H\2\2cd\7C\2\2de\7N\2\2ef\7U\2\2fh\7G\2\2g]\3\2"+
+		"\2\2gb\3\2\2\2h\f\3\2\2\2ij\7?\2\2jk\7?\2\2k\16\3\2\2\2lm\7#\2\2mn\7?"+
+		"\2\2n\20\3\2\2\2op\7>\2\2p\22\3\2\2\2qr\7>\2\2rs\7?\2\2s\24\3\2\2\2tu"+
+		"\7@\2\2u\26\3\2\2\2vw\7@\2\2wx\7?\2\2x\30\3\2\2\2yz\7.\2\2z\32\3\2\2\2"+
+		"{|\7]\2\2|\34\3\2\2\2}~\7_\2\2~\36\3\2\2\2\177\u0080\7*\2\2\u0080 \3\2"+
+		"\2\2\u0081\u0082\7+\2\2\u0082\"\3\2\2\2\u0083\u0084\7k\2\2\u0084\u0085"+
+		"\7p\2\2\u0085$\3\2\2\2\u0086\u0087\7p\2\2\u0087\u0088\7q\2\2\u0088\u0089"+
+		"\7v\2\2\u0089\u008a\7\"\2\2\u008a\u008b\7k\2\2\u008b\u008c\7p\2\2\u008c"+
+		"&\3\2\2\2\u008d\u008e\7g\2\2\u008e\u008f\7z\2\2\u008f\u0090\7k\2\2\u0090"+
+		"\u0091\7u\2\2\u0091\u0092\7v\2\2\u0092\u0093\7u\2\2\u0093(\3\2\2\2\u0094"+
+		"\u0096\4\62;\2\u0095\u0094\3\2\2\2\u0096\u0097\3\2\2\2\u0097\u0095\3\2"+
+		"\2\2\u0097\u0098\3\2\2\2\u0098*\3\2\2\2\u0099\u009b\4\62;\2\u009a\u0099"+
+		"\3\2\2\2\u009b\u009c\3\2\2\2\u009c\u009a\3\2\2\2\u009c\u009d\3\2\2\2\u009d"+
+		"\u009e\3\2\2\2\u009e\u00a0\7\60\2\2\u009f\u00a1\4\62;\2\u00a0\u009f\3"+
+		"\2\2\2\u00a1\u00a2\3\2\2\2\u00a2\u00a0\3\2\2\2\u00a2\u00a3\3\2\2\2\u00a3"+
+		",\3\2\2\2\u00a4\u00a8\t\2\2\2\u00a5\u00a7\t\3\2\2\u00a6\u00a5\3\2\2\2"+
+		"\u00a7\u00aa\3\2\2\2\u00a8\u00a6\3\2\2\2\u00a8\u00a9\3\2\2\2\u00a9.\3"+
+		"\2\2\2\u00aa\u00a8\3\2\2\2\u00ab\u00ac\n\4\2\2\u00ac\60\3\2\2\2\u00ad"+
+		"\u00b1\7$\2\2\u00ae\u00b0\5/\30\2\u00af\u00ae\3\2\2\2\u00b0\u00b3\3\2"+
+		"\2\2\u00b1\u00af\3\2\2\2\u00b1\u00b2\3\2\2\2\u00b2\u00b4\3\2\2\2\u00b3"+
+		"\u00b1\3\2\2\2\u00b4\u00be\7$\2\2\u00b5\u00b9\7)\2\2\u00b6\u00b8\5/\30"+
+		"\2\u00b7\u00b6\3\2\2\2\u00b8\u00bb\3\2\2\2\u00b9\u00b7\3\2\2\2\u00b9\u00ba"+
+		"\3\2\2\2\u00ba\u00bc\3\2\2\2\u00bb\u00b9\3\2\2\2\u00bc\u00be\7)\2\2\u00bd"+
+		"\u00ad\3\2\2\2\u00bd\u00b5\3\2\2\2\u00be\62\3\2\2\2\u00bf\u00c0\7=\2\2"+
+		"\u00c0\64\3\2\2\2\u00c1\u00c2\7\61\2\2\u00c2\u00c3\7\61\2\2\u00c3\u00c5"+
+		"\3\2\2\2\u00c4\u00c6\13\2\2\2\u00c5\u00c4\3\2\2\2\u00c6\u00c7\3\2\2\2"+
+		"\u00c7\u00c8\3\2\2\2\u00c7\u00c5\3\2\2\2\u00c8\u00ca\3\2\2\2\u00c9\u00cb"+
+		"\t\5\2\2\u00ca\u00c9\3\2\2\2\u00cb\u00cc\3\2\2\2\u00cc\u00cd\b\33\2\2"+
+		"\u00cd\66\3\2\2\2\u00ce\u00d0\t\6\2\2\u00cf\u00ce\3\2\2\2\u00d0\u00d1"+
+		"\3\2\2\2\u00d1\u00cf\3\2\2\2\u00d1\u00d2\3\2\2\2\u00d2\u00d3\3\2\2\2\u00d3"+
+		"\u00d4\b\34\2\2\u00d48\3\2\2\2\22\2AIQ[g\u0097\u009c\u00a2\u00a8\u00b1"+
+		"\u00b9\u00bd\u00c7\u00ca\u00d1\3\b\2\2";
```

Comment:
Because this is generated code, people just cloning the repo and spinning it up in an IDE would have to either configure their IDE to run the maven generate-sources target or externally run maven to get the code to compile.  I think that generated code that is rarely touched existing in the SCM is generally better than instructing people to run maven before they start their IDEs or, worse, describe how to set up their IDEs to generate the sources.

I'm open to a discussion, obviously.